### ISSUE TITLE
Same original behaviour with new firmware

### ISSUE THREAD

**Issue by zimgui** [Andrik45719/ZY-M100#10]
I have 6 ZY-M100-24GV3_TZE204_ya4ft0w4, 3 ceiling and 3 wall mounted, I flashed all of them with your new firmware ZY-M100-24GV3_TZE204_ya4ft0w4_increased_report_interval.bin using your guide and initially seems to work fine but in a few moments all of them begin to send messages to the zigbee network again every second.
Important, seems that they become quiet and stop sending messages every second when presence or move is detected.
Any idea that what I'm doing wrong?
Apologies for my english and thank you very much for your work.
Pd: I tried your other firmware without increased report interval with the same result.

**Comment by bundoon** [comment]
I'm getting the same zimgui.. still very chatty after flashing new firmware ZY-M100-24GV3_TZE204_ya4ft0w4_increased_report_interval.bin. Turned off distance switch too in zigbee2mqtt.

**Comment by zimgui** [comment]
Sure, I send the captured log of 5 devices where adresses from the devices are replaced with 111..., 222...
[tuya.log](https://github.com/user-attachments/files/17463868/tuya.log)
Let me know if you need more or something else, thanks!

**Comment by Andrik45719** [comment]
try this one with disabled Zero Target Distance reporting
ZY-M100-24GV3_TZE204_ya4ft0w4_increased_report_interval_TD_0_disable.bin

**Comment by zimgui** [comment]
I flashed a new one and seems that works fine, let me check it in a few hours, I tried to flash the previous devices already flashed with the previous vertion and not able to deploy the new firmware, always getting this error:

Open On-Chip Debugger 0.12.0 (2023-01-14-23:37)
Licensed under GNU GPL v2
For bug reports, read
        http://openocd.org/doc/doxygen/bugs.html
Info : auto-selecting first available session transport "hla_swd". To override use 'transport select <transport>'.
Info : The selected transport took over low-level target control. The results might differ compared to plain JTAG/SWD
Info : clock speed 1000 kHz
Info : STLINK V2J37S7 (API v2) VID:PID 0483:3748
Info : Target voltage: 3.270120
Error: init mode failed (unable to connect to the target)_

Any idea?
